qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -205,56 +205,65 @@ def spin2(electrons, orbitals, mapping="jordan_wigner", wires=None):
-    if any, not included in the active space and
-    :math:`\langle \alpha \vert \hat{\mathcal{A}} \vert \beta \rangle` and
-    :math:`\langle \alpha, \beta \vert\hat{\mathcal{B}} \vert \gamma, \delta \rangle`
-    are the matrix elements of the one- and two-particle operators
-    :math:`\hat{\mathcal{A}}` and :math:`\hat{\mathcal{B}}`, respectively.
+        \hat{O} = \sum_{\alpha, \beta} \langle \alpha \vert \hat{t}^{(1)} + \hat{t}^{(2)}
+        \cdots + \hat{t}^{(n)} \vert \beta \rangle ~ \hat{c}_\alpha^\dagger \hat{c}_\beta
+        + \frac{1}{2} \sum_{\alpha, \beta, \gamma, \delta}
+        \langle \alpha, \beta \vert \hat{v}^{(1)} + \hat{v}^{(2)} \cdots + \hat{v}^{(n)}
+        \vert \gamma, \delta \rangle ~ \hat{c}_\alpha^\dagger \hat{c}_\beta^\dagger
+        \hat{c}_\gamma \hat{c}_\delta
+
+    In the latter equations the indices :math:`\alpha, \beta, \gamma, \delta` run over the
+    basis of single-particle states. The operators :math:`\hat{c}^\dagger` and :math:`\hat{c}`
+    are the particle creation and annihilation operators, respectively.
+    :math:`\langle \alpha \vert \hat{t} \vert \beta \rangle` denotes the matrix element of
+    the single-particle operator :math:`\hat{t}` entering the observable. For example,
+    in electronic structure calculations this is the case for the kinetic energy operator,
+    the nuclei Coulomb potential or any other external fields included in the model Hamiltonian.
+    On the other hand, :math:`\langle \alpha, \beta \vert \hat{v} \vert \gamma, \delta \rangle`
+    denotes the matrix element of the two-particle operator :math:`\hat{v}`, for example, the
+    Coulomb interaction between the electrons.
+
+    If an `active space <https://en.wikipedia.org/wiki/Complete_active_space>`_
+    (see :func:`~.active_space`) is defined, the observable is expanded over the truncated
+    basis of active orbitals. The contribution of core orbitals, if any, can be passed to the
+    function using the keyword argument ``init_term``.
 
     The function utilizes tools of `OpenFermion <https://github.com/quantumlib/OpenFermion>`_
     to build the second-quantized operator and map it to basis of Pauli matrices via the
     Jordan-Wigner or Bravyi-Kitaev transformation. Finally, the qubit operator is
     converted to a a PennyLane observable by the function :func:`~.convert_observable`.
 
     Args:
-        me_table (array[float]): Numpy array with the table of matrix elements.
-            For single-particle operators this array will have shape
-            ``(me_table.shape[0], 3)`` with each row containing the indices
-            :math:`\alpha`, :math:`\beta` and the matrix element :math:`\langle \alpha \vert
-            \hat{\mathcal{A}}\vert \beta \rangle`. For two-particle operators this
-            array will have shape ``(me_table.shape[0], 5)`` with each row containing
+        matrix_elements (list(array[float])): list containing 2D numpy arrays with the matrix
```

Comment:
Why not pass two tables, one for the single particle contribution and another for the two-particle one? 
I find it a bit clunky and heavy to bundle them together in a big list. We could set the two-particle one to `None` by default, so that for a single-particle observable we only pass that table, and for the two-particle one, we pass both.

Reply:
So the UI would be something like:
**single-particle**
`observable(t)`

**two-particle**
`observable(t, v)`

where I'm using the defaults for `init_term` and `mapping`